client/src/Constants.js

```diff
@@ -0,0 +1,10 @@
+/* Store constant values here */
+
+export const minPasswordLength = 6;
+export const minPasswordErrMsg = `Your password must be at least ${minPasswordLength} characters long`;
+export const takenEmailErrMsg = "That email is already taken";
+
+const serverUrl = "http://localhost:5000/";
+const clientUrl = "http://localhost:3000/";
+export const serverSignUpUrl = serverUrl + "users";
```

Comment:
It will just be user, singular for the url route, since the operations we're doing will only be affecting one user at a time, and currently I don't think we have any plans for operations that will affect multiple users.

Reply:
made the change

Reply:
did you commit/push it? Can't see the change yet.

Reply:
No I didn't yet. I made a bunch of other changes (not usable atm) so committing now will be inconvenient.

Reply:
Probably a good idea if you start working on other features unrelated to this PR to put them in a separate branch.

My manager when I worked at cineplex used to tell me that a PR should represent one finished piece of work/feature. (I was really bad about that, I'd get carried away and have like 3 features done in one PR...)

Reply:
Well the changes I'm making are related to this PR because the changes have been made to the login page which is so similar to the sign up page. But I just realized the name so I guess I'll have to undo the changes and commit and merge.

Reply:
I'd say don't worry about it this time, and just keep it in mind for the future

Reply:
Its alright. I didn't lose any code. new and final PR coming up